auto-value-moshi/src/test/java/com/ryanharter/auto/value/moshi/AutoValueMoshiExtensionTest.java

```diff
@@ -110,8 +110,8 @@
         + "    private final JsonAdapter<Integer> cAdapter;\n"
         + "    private final JsonAdapter<String> dAdapter;\n"
         + "    private final JsonAdapter<Map<String, Number>> eAdapter;\n"
-        + "    private final JsonAdapter<Set<String>> fAdapter;\n"
-        + "    private final JsonAdapter<Map<String, Set<String>>> gAdapter;\n"
+        + "    private final JsonAdapter<Set<? extends String>> fAdapter;\n"
```

Comment:
Moshi’s `adapter()` method below trusts you to get your types right. But here you’re slightly cheating, coercing a `Set<String>` to a `Set<? extends String>`. In practice it’s the same, but it isn’t guaranteed in general.

Might be better to just add the `Types.subtypeOf()`.